### DISCUSSION TITLE
Does expo-image-picker include all permissions by default for iOS and Android?

### DISCUSSION THREAD

**Discussion by ibrahim-ytl** [expo/expo#31049] [Archived: Questions] [answered]
Does `expo-image-picker` include all permission by default for iOS and Android?

Permission such as:
- `microphonePermission`
- `cameraPermission`
- `photoPermission`

If yes, how can I remove all this permissions?

**Comment by tsapeta** [comment]
You can use its config plugin, see https://docs.expo.dev/versions/latest/sdk/imagepicker/#example-appjson-with-config-plugin
You can provide custom permission descriptions or set them to `false` if you want them to be deleted during `expo prebuild` command.

**Comment by ibrahim-ytl** [comment]
Hey, thanks for the quick response. In my case, I already set it to false:

![Screenshot 2024-08-19 at 5 13 51 PM](https://github.com/user-attachments/assets/979a1dd4-55ec-4d21-b9b3-02b4fb1acd20)

But I can still see the permission in my info plist after prebuild:

![Screenshot 2024-08-19 at 5 13 59 PM](https://github.com/user-attachments/assets/6787a8f4-5099-47f3-9ecc-16fd4046895d)

**Comment by ibrahim-ytl** [comment, marked as answer]
Turns out it was a mistake from my side. I missed "s" from "photos". Instead I wrote `photo`. Hence it should be `photosPermission` instead of `photoPermission`.
